recipes/butteraugli/all/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 3.4)
```

Comment:
I think we need a bazel recipe + better conan bazel support.
Because right now, we need to create cmake script for all google projects.

Reply:
> better conan bazel support

I tried to write a bazel generator but bazel doesn't support absolute paths. Bazel is designed to handle dependencies itself, I don't think it's possible to have bazel support for conan.

Reply:
bazel must have some way to consume e.g. .pc files?

Reply:
I've opened https://github.com/conan-io/conan-center-index/issues/2897 as a reminder.